package.json

```diff
@@ -35,7 +35,7 @@
     "prettier": "^2.0.1",
     "replace-in-files-cli": "^1.0.0",
     "ts-jest": "^26.2.0",
-    "typescript": "^4.0.5"
+    "typescript": "4.0.3"
```

Comment:
I've pinned the version of typescript because TS `4.1+` is not working with ts-jest and causing tests to fail.

Officially, the version `4.0+` is not supported by ts-jest, but `4.0.x` they are working on this. Until it's resolved upstream, we should keep this pinned.

Here's the ts-jest issue: https://github.com/kulshekhar/ts-jest/issues/1887